AudioTranscriber.py

```diff
@@ -93,6 +93,8 @@ def update_transcript(self, who_spoke, text, time_spoken):
             transcript.insert(0, (f"{who_spoke}: [{text}]\n\n", time_spoken))
         else:
             transcript[0] = (f"{who_spoke}: [{text}]\n\n", time_spoken)
+    
```

Comment:
blank space added?

Reply:
Sry meant to say that you added extra black line here unnecessary. Please remove this change